package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unstoppabledomains/ui-kit",
-  "version": "0.2.3",
+  "version": "0.2.4-alpha.0",
```

Comment:
Should we simply name it 0.2.4?

Reply:
Yep, it'll become 0.2.4 once these changes are stabilized and ready to be merged. `-alpha.*` is used for incrementing test releases indicating that they're not stable yet. Also, it allows us to publish a version to npm and use the new version in the [website's PR](https://github.com/unstoppabledomains/unstoppable-domains-website/pull/5840) without breaking the checks.